pyt/github_search.py

```diff
@@ -209,7 +210,7 @@ def scan_github(search_string, start_date, analysis_type, analyse_repo_func, csv
                       Languages.python, repo)
             s = SearchCode(q)
             if s.results:
-                r = repo_runner.Repo(repo.url)
+                r = Repo(repo.url)
```

Comment:
This Repo is referencing the definition of Repo in this module and not the one in the repo_runner.